c7n/resources/cloudfront.py

```diff
@@ -149,6 +150,55 @@ def process(self, resources, event=None):
         return results
 
 
+@Distribution.filter_registry.register('distribution-config')
+class DistributionConfig(ValueFilter):
+    """Check for Cloudfront distribution config values
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: logging-enabled
+                resource: distribution
+                filters:
+                  - type: distribution-config
+                    key: Logging.Enabled
+                    value: true
+   """
+
+    schema = type_schema('distribution-config', rinherit=ValueFilter.schema)
+    schema_alias = False
+    permissions = ('cloudfront:GetDistributionConfig',)
+    annotation_key = 'c7n:distribution-config'
+    annotate = False
+
+    def process(self, resources, event=None):
+
+        self.augment([r for r in resources if self.annotation_key not in r])
+        return super(DistributionConfig, self).process(resources, event)
```

Comment:
more fwiw, in py3, this can be just done as super().process(resources, event), the longer form form is for py2 compatibility which is no longer being maintained.